src/app/profile/yourGiving/stopStartRecurringGifts/restartGift/step0/addUpdatePaymentMethod.component.js

```diff
@@ -0,0 +1,86 @@
+import angular from 'angular';
```

Comment:
Alrighty. I think that's fine. If you have some time in the future maybe we could consolidate them.